components/Element/ChatbotLinkItem.vue

```diff
@@ -33,7 +45,28 @@ defineProps({
         required: true,
     },
     indexed: Boolean,
+    collectionid: String,
+    httperror: String,
 });
-
 const editMode = ref(false);
+const indexing = ref(false)
+
+const reindex = async(url: string, collection: string | undefined, metadata: any) => {
+    indexing.value = true;
+    try {
+        await $fetch(`/api/brevia/index/link`, {
+            method: 'POST',
+            body: {
+                link: url,
+                collection_id : collection,
+                metadata: metadata,
+                options: {},
+            },
+        });
+        indexing.value = false;
```

Comment:
after a reindex we should read the indexed document in order to see if a document has been actually indexed/updated or not  

We should call `GET /api/brevia/index/{collection_id}/{document_id}` or `GET /api/brevia/index/{collection_id}/document_metadata?document_id={document_id}`

A user feedback is then needed + change of state (like http_error removal) if something has changed